roboflow/core/version.py

```diff
@@ -136,6 +140,13 @@ def __init__(
             else:
                 self.model = None
 
+            # check if version has a model
+            if version_info.status_code == 200:
+                version_info = version_info.json()["version"]
+
+                if not bool(version_info["models"]):
```

Comment:
`if` calls `bool` for you. You don't need to do it explicitly.

Reply:
🤦 yeah